packages/parse-cli-args/test/index.ts

```diff
@@ -196,10 +195,24 @@ test('use command-specific shorthands', async (t) => {
 test('any unknown command is treated as a script', async (t) => {
   const { options, cmd, params } = await parseCliArgs({
     ...DEFAULT_OPTS,
+    fallbackCommand: 'run',
+    getCommandLongName: () => null,
     universalOptionsTypes: { filter: [String, Array] },
   }, ['foo', '--recursive'])
   t.equal(cmd, 'run')
   t.deepEqual(params, ['foo'])
   t.ok(options['recursive'])
   t.end()
 })
+
+test("don't use the fallback command if no command is present", async (t) => {
+  const { cmd, params } = await parseCliArgs({
+    ...DEFAULT_OPTS,
+    fallbackCommand: 'run',
+    getCommandLongName: () => null,
+    universalOptionsTypes: { filter: [String, Array] },
+  }, [])
+  t.equal(cmd, null)
+  t.deepEqual(params, [])
+  t.end()
+})
```

Comment:
**Nit:** This is missing a trailing newline.
```suggestion
})
```

Reply:
There was no trailing new line. I believe the linter currently wants no newline at the end